tests/unit/plugins/filter/test_find_template_roles.py

```
@@ -271,6 +275,50 @@ def test_network_has_capabilities(self, roles_dir):
+        assert metadata.defaults is not None
+        assert metadata.defaults.virtual_network_ipv4_cidr == "10.200.0.0/16"
+        assert metadata.defaults.subnet_ipv4_cidr == "10.200.0.0/20"
+        assert metadata.defaults.enable_nat_gateway is False
+        assert metadata.defaults.egress_rules is not None
+        assert len(metadata.defaults.egress_rules) == 1
+        assert metadata.defaults.egress_rules[0].protocol == "PROTOCOL_ALL"
+
+    def test_network_defaults_serialized_in_spec(self):
+        template = NetworkClassTemplate(
+            collection="test",
+            path=Path("/tmp/test"),
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)